src/new/wallet.js

```diff
@@ -316,15 +316,16 @@ class HathorWallet extends EventEmitter {
    * @param {String} address Address to send the tokens
    * @param {number} value Amount of tokens to be sent
    * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
+   * @param {String} changeAddress Optional parameter with address for the change output
    *
    * @return {Object} In case of success, an object with {success: true, sendTransaction, promise}, where sendTransaction is a
    * SendTransaction object that emit events while the tx is being sent and promise resolves when the sending is done
    * In case of error, an object with {success: false, message}
    *
    **/
-  sendTransaction(address, value, token) {
+  sendTransaction(address, value, token, changeAddress = null) {
```

Comment:
I'd use a parameters called options that should receive an object. It will allow simple future extension with more options. So, it would be `sendTransaction(address, value, token, { changeAddress = null }) {...}`.

Reply:
Done but the method signature ended up being a bit different.
```
sendTransaction(address, value, token, options = { changeAddress: null }) {
```

What do you think?

Reply:
Great! Thanks!